src/thebelab.js

```diff
@@ -424,34 +433,69 @@ export function requestBinder({
   }
   console.log("Binder build URL", url);
 
-  return new Promise(async (resolve, reject) => {
-    // if binder already spawned our pod and we remember the creds, reuse it
-    let existing_server = JSON.parse(
-      window.localStorage.getItem("thebe-binder-" + url)
+  const storageKey = savedSession.storagePrefix + url;
+
+  async function getExistingServer() {
+    if (!savedSession.enabled) {
+      return;
+    }
+    let storedInfoJSON = window.localStorage.getItem(storageKey);
+    if (storedInfoJSON == null) {
+      console.debug("No session saved in ", storageKey);
+      return;
+    }
+    console.debug("Saved binder session detected");
+    let existingServer = JSON.parse(storedInfoJSON);
+    let timestamp = new Date(existingServer.timestamp);
+    if (new Date().getTime() - timestamp.getTime() < savedSession.maxAge) {
```

Comment:
1. The if statement should trigger if the time difference is _bigger_ than the max age
2. This is my fault, but upon googling the `getTime()` function returns milliseconds rather than seconds, so we should probably divide by 1000